src/plugins/workspace/public/components/workspace_initial/workspace_use_case_card.scss

```diff
@@ -0,0 +1,80 @@
+/* stylelint-disable */
+.observability {
+  background-color: hsla(30deg, 24%, 96%, 100%);
+  background-image:
```

Comment:
Can we move the customized gradient to a file like gradient_variables.scss? So that we only apply the stylelint-disable rule in that file.

Reply:
Sure, I will update it.